protocols/migration/migration_reports_4_0_0_to_reports_4_2_0.py

```diff
@@ -0,0 +1,237 @@
+from protocols import reports_4_0_0 as reports_4_0_0
+from protocols import reports_4_2_0
+from protocols.migration import BaseMigration
+from protocols.migration.participants import MigrationParticipants100To104
+
+
+class MigrateReports400To420(BaseMigration):
+
+    old_model = reports_4_0_0
+    new_model = reports_4_2_0
+
+    def migrate_cancer_interpretation_request(self, cancer_interpretation_request, sample_id):
+        """
+        :type cancer_interpretation_request: reports_4_0_0.CancerInterpretationRequest
+        :type sample_id: str
+        :rtype: reports_4_2_0.CancerInterpretationRequest
+        """
+        new_cancer_interpretation_request = self.new_model.CancerInterpretationRequest.fromJsonDict(
+            jsonDict=cancer_interpretation_request.toJsonDict()
+        )
+
+        new_cancer_interpretation_request.gitVersionControl = "4.2.0"
+
+        new_cancer_participant = MigrationParticipants100To104().migrate_cancer_participant(
+            cancer_participant=cancer_interpretation_request.cancerParticipant
+        )
+        new_cancer_interpretation_request.cancerParticipant = new_cancer_participant
+
+        # FIXME: we are using a sample_id received as a parameter, models need to be changed
```

Comment:
@priesgo do we need to change 4.2.0 before closing it ?

Reply:
no